examples/ray_plugin/ray_plugin/ray_example.py

```diff
@@ -33,6 +23,7 @@
     name="ray-flyte-plugin",
     registry="ghcr.io/flyteorg",
     packages=["flytekitplugins-ray"],
+    apt_packages=["wget"],
```

Comment:
add a comment to say why we need `wget`?

Reply:
Done